arkouda/random/_legacy.py

```diff
@@ -147,11 +149,23 @@ def standard_normal(size: int_scalars, seed: Union[None, int_scalars] = None) ->
     >>> ak.standard_normal(3,1)
     array([-0.68586185091150265, 1.1723810583573375, 0.567584107142031])
     """
-    if size < 0:
-        raise ValueError("The size parameter must be > 0")
+    shape: Union[int_scalars, Tuple[int_scalars, ...]] = 1
+    if isinstance(size, tuple):
+        shape = cast(Tuple, size)
+        full_size = 1
+        for s in cast(Tuple, shape):
+            full_size *= s
+        ndim = len(shape)
+    else:
+        full_size = cast(int, size)
+        if full_size < 0:
+            raise ValueError("The size parameter must be > 0")
```

Comment:
@ajpotts good question! thanks for checking but this is intentional. The argument passed into the function is `size` (which is used to create `full_size`) so that's what the user would need to change to fix the error